tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -158,9 +158,9 @@ kind: Pod
 metadata:
   annotations:
     openshift.io/scc: privileged
-  generateName: ovs-
+  generateName: ovsdbserver-sb-
   labels:
-    service: ovs
+    service: ovsdbserver-sb
```

Comment:
@dprince I think if we switch these back, the KUTTL test will pass now.  I tried it locally with a clone of your PR and it worked.

Reply:
@abays thanks for that. I will remove the offending lines.